src/Pages/SpotifyCallbackPage/SpotifyCallbackPage.js

```diff
@@ -0,0 +1,31 @@
+import { useEffect } from "react";
+import { Navigate } from "react-router";
+import useAuthentication from "Services/firebase/useAuthentication";
+import useSetSpotifyAuthData from "Store/updaters/useSetSpotifyAuthData";
+import useHashParams from "Utils/useHashParams";
+
+const SpotifyCallbackPage = function () {
+  const { authReady, loggedIn } = useAuthentication();
+  const params = useHashParams();
+  const setSpotifyAuthData = useSetSpotifyAuthData();
+
+  useEffect(() => {
+    if (authReady && loggedIn && params.get("access_token")) {
+      setSpotifyAuthData({
+        access_token: params.get("access_token"),
+        token_type: params.get("token_type"),
+        expires: Date.now() + (parseInt(params.get("expires_in") * 1000))
+      });
+    }
+  }, [authReady, loggedIn, params, setSpotifyAuthData]);
+
+  if (!authReady)
+    return <span>Waiting for auth...</span>;
```

Comment:
Later we might have to insert some Gif or so for the "Feedback of Action" grading criteria

Reply:
Yep, exactly. Creating some nice looking things could be part of #24